<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configuration</class>
 <widget class="QWidget" name="configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>747</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name">
       <property name="placeholderText">
        <string>Global name to this calculation (e.g. &quot;Kensai/9 Ashideena and Belm&quot;)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="commonGroup">
     <property name="title">
      <string>Common</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="baseThac0Label">
          <property name="text">
           <string>THAC0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="baseThac0">
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="strengthThac0BonusLabel">
          <property name="text">
           <string>Strength bonus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="strengthThac0Bonus"/>
        </item>
        <item>
         <widget class="QLabel" name="classThac0BonusLabel">
          <property name="text">
           <string>Class bonus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="classThac0Bonus"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="strengthDamageBonusLabel">
          <property name="text">
           <string>Damage Strength bonus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="strengthDamageBonus"/>
        </item>
        <item>
         <widget class="QLabel" name="classDamageBonusLabel">
          <property name="text">
           <string>Damage class bonus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="classDamageBonus"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="maximumDamage">
        <property name="text">
         <string>Maximum damage (Kai, Righteous Magic, etc.)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QGroupBox" name="mainHandGroup">
       <property name="title">
        <string>Main Hand</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="mainHandThac0Group">
          <property name="title">
           <string>THAC0</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="proficiencyThac0Modifier1Label">
             <property name="text">
              <string>Proficency modifier</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="proficiencyThac0Modifier1">
             <property name="minimum">
              <number>-5</number>
             </property>
             <property name="maximum">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="styleModifier1Label">
             <property name="text">
              <string>Style modifier</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="styleModifier1">
             <property name="minimum">
              <number>-4</number>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="weaponThac0Modifier1Label">
             <property name="text">
              <string>Weapon modifer</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="weaponThac0Modifier1">
             <property name="maximum">
              <number>6</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="mainHandDamageGroup">
          <property name="title">
           <string>Damage</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="proficiencyDamageModifier1Label">
               <property name="text">
                <string>Proficency modifier</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="proficiencyDamageModifier1">
               <property name="minimum">
                <number>-1</number>
               </property>
               <property name="maximum">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="damageDice1Layout">
             <item row="0" column="1">
              <widget class="QSpinBox" name="weaponDamageDiceNumber1">
               <property name="suffix">
                <string> dice</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="weaponDamageDiceSide1">
               <property name="suffix">
                <string> sides</string>
               </property>
               <property name="minimum">
                <number>4</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="weaponDamageDiceBonus1">
               <property name="suffix">
                <string> bonus</string>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="weaponDamageDiceLabel1">
               <property name="text">
                <string>Weapon</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="damageType1Label">
               <property name="text">
                <string>Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="damageType1">
               <item>
                <property name="text">
                 <string>Crushing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Missile</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Piercing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Slashing</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="damageDetail1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="attacksPerRound1Label">
            <property name="text">
             <string>Attacks Per Round</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="attacksPerRound1">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="doubleCriticalChance">
          <property name="text">
           <string>Critical hit on 19</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="offHandGroup">
       <property name="title">
        <string>Off-hand</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="offHandThac0Group">
          <property name="title">
           <string>THAC0</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="proficiencyThac0Modifier2Label">
             <property name="text">
              <string>Proficency modifier</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="proficiencyThac0Modifier2">
             <property name="minimum">
              <number>-5</number>
             </property>
             <property name="maximum">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="styleModifier2Label">
             <property name="text">
              <string>Style modifier</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="styleModifier2">
             <property name="minimum">
              <number>-8</number>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="weaponThac0Modifier2Label">
             <property name="text">
              <string>Weapon modifer</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="weaponThac0Modifier2">
             <property name="maximum">
              <number>6</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="offHandDamageGroup">
          <property name="title">
           <string>Damage</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="proficiencyDamageModifier2Label">
               <property name="text">
                <string>Proficency modifier</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="proficiencyDamageModifier2">
               <property name="minimum">
                <number>-1</number>
               </property>
               <property name="maximum">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="damageDice2Layout">
             <item row="0" column="1">
              <widget class="QSpinBox" name="weaponDamageDiceNumber2">
               <property name="suffix">
                <string> dice</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="weaponDamageDiceSide2">
               <property name="suffix">
                <string> sides</string>
               </property>
               <property name="minimum">
                <number>4</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="weaponDamageDiceBonus2">
               <property name="suffix">
                <string> bonus</string>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="weaponDamageDiceLabel2">
               <property name="text">
                <string>Weapon</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="damageType2Label">
               <property name="text">
                <string>Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="damageType2">
               <item>
                <property name="text">
                 <string>Crushing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Missile</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Piercing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Slashing</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="damageDetail2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="attacksPerRound2Label">
            <property name="text">
             <string>Attacks Per Round</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="attacksPerRound2">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
